astropy/modeling/tests/test_fix_inputs.py

```diff
@@ -0,0 +1,248 @@
+import numpy as np
```

Comment:
Why is this a new test module? There are many other existing tests for `fix_inputs` in other modules, meaning this separation at this time is a big misleading. While I do think that it may be worth reorganizing the tests to have a module like  this I think that should be done in a separate PR.  I suggest placing this in `test_fitters`, `test_compound`, and/or `test_core`.